location/templates/location.html

```diff
@@ -22,5 +23,41 @@
 
 {% block content %}
 <h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
+    # After onclick "write review...",  should POST and display write review
+    # should automatically update the page
 {% leaflet_map "leaflet_map_main" %}
+    {% if object.review_set %}
+        <h2> Review: </h2>
+        <hr class="mt-0 mb-2 mt-2">
+        <div class="row h-65">
+            <div class="col">
+                <table class="table table-striped table-bordered table-hover table-sm results-table" width="100%">
+                    <thead>
+                        <tr>
+                            <th scope="col" class="th-sm">User</th>
+                            <th scope="col" class="th-sm">Content</th>
+                            <th scope="col" class="th-sm">Time</th>
```

Comment:
would it be possible to shrink the User and time columns? It doesn't make sense to have each take up the same amount of space as the entire review. The review column should be the majority.

Reply:
Revised